openelevationservice/server/api/elevation_query_parallel.py

```diff
@@ -0,0 +1,152 @@
+from sqlalchemy import text
+import json
+from shapely.geometry import shape, mapping, Polygon, MultiPolygon
+from shapely.ops import unary_union
+from multiprocessing import Pool
+import math
+
+#Start-Query code
+query = text(
+    """
+    WITH query_geom AS (
+    SELECT ST_SetSRID(ST_GeomFromText(:polygon), 4326) AS geom
+),
+polygons AS (
+    SELECT *
+    FROM (
+        SELECT 
+            (ST_PixelAsPolygons(
+                ST_Clip(oes_cgiar.rast, query_geom.geom, 0),
+                1, False
+            )).geom AS geometry,  -- Extrae las celdas como geometría
```

Comment:
There can be no commentary in Spanish. Translate it.

Reply:
Done. Deleted.